examples/table/table.bal

```diff
@@ -96,23 +96,21 @@ public function main() {
     io:println(tb);
 
     // This converts the table to JSON format.
-    var retValJson = <json>tb;
-    match retValJson {
-        json j => {
-            io:print("JSON: ");
-            io:println(j);
-        }
-        error e => io:println("Error in table to json conversion");
+    var retValJson = json.convert(tb);
+    if (retValJson is json) {
+        io:print("JSON: ");
```

Comment:
Why can't we write this in a single line?

Reply:
Will bring into one line.